README.md

```diff
@@ -21,7 +21,7 @@
 
     ```
     $ pip install -r requirements/common.txt
-    $ pip install -r requirements/dev.txt
+    $ pip install -r requirements/dev.  txt
```

Comment:
No problem. I would just reset this file. You can do that with git [read here](https://github.com/blog/2019-how-to-undo-almost-anything-with-git) or much easier just delete the space int he file and recommit